sdk/batch/Azure.Compute.Batch/tests/Integration/BatchNodeIntegrationTests.cs

```diff
@@ -0,0 +1,255 @@
+                    Assert.NotNull(batchNodeVMExtension);
+                }
+            }
+            finally
+            {
+                await client.DeletePoolAsync(poolID);
+            }
+        }
+
+        [RecordedTest]
+        public async Task GetRemoteLoginSettings()
+        {
+            var client = CreateBatchClient();
+            IaasLinuxPoolFixture iaasWindowsPoolFixture = new IaasLinuxPoolFixture(client, "GetRemoteLoginSettings", isPlayBack());
+            var poolID = iaasWindowsPoolFixture.PoolId;
+
+            try
+            {
+                // create a pool to verify we have something to query for
+                BatchPool pool = await iaasWindowsPoolFixture.CreatePoolAsync(2);
+
+                string batchNodeID = "";
+                await foreach (BatchNode item in client.GetNodesAsync(poolID))
+                {
+                    batchNodeID = item.Id;
+                }
+                Assert.IsNotEmpty(batchNodeID);
+
+                BatchNodeRemoteLoginSettings batchNodeRemoteLoginSettings = await client.GetNodeRemoteLoginSettingsAsync(poolID, batchNodeID);
+                Assert.NotNull(batchNodeRemoteLoginSettings);
+                Assert.IsNotEmpty(batchNodeRemoteLoginSettings.RemoteLoginIpAddress);
+            }
+            finally
+            {
+                await client.DeletePoolAsync(poolID);
+            }
+        }
+
+        [RecordedTest]
+        public async Task Scheduling()
+        {
+            var client = CreateBatchClient();
+            IaasLinuxPoolFixture iaasWindowsPoolFixture = new IaasLinuxPoolFixture(client, "Scheduling", isPlayBack());
+            var poolID = iaasWindowsPoolFixture.PoolId;
+
+            try
+            {
+                // create a pool to verify we have something to query for
+                BatchPool pool = await iaasWindowsPoolFixture.CreatePoolAsync(1);
+
+                string batchNodeID = "";
+                await foreach (BatchNode item in client.GetNodesAsync(poolID))
+                {
+                    batchNodeID = item.Id;
+                }
+                Assert.IsNotEmpty(batchNodeID);
+                BatchNodeDisableSchedulingContent batchNodeDisableSchedulingContent = new BatchNodeDisableSchedulingContent()
+                {
+                    NodeDisableSchedulingOption = BatchNodeDisableSchedulingOption.TaskCompletion,
+                };
+                Response response = await client.DisableNodeSchedulingAsync(poolID, batchNodeID, batchNodeDisableSchedulingContent);
+                Assert.AreEqual(200, response.Status);
+
+                response = await client.EnableNodeSchedulingAsync(poolID, batchNodeID);
+                Assert.AreEqual(200, response.Status);
+
+                UploadBatchServiceLogsContent uploadBatchServiceLogsContent = new UploadBatchServiceLogsContent("http://fake.com", DateTimeOffset.Parse("2026-05-01T00:00:00.0000000Z"));
```

Comment:
This really doesn't matter, but it's nice to use contoso.com for fake domains since Microsoft owns it (on the off-chance that buggy test code makes a real request).

Reply:
updated